src/api/app/views/webui2/webui/request/_request_history.html.haml

```diff
@@ -0,0 +1,34 @@
+%ul.list-unstyled
+  %li.media
+    - user = User.find_by_login(bs_request.creator)
+    = image_tag_for(user, size: 32, custom_class: 'mr-3', width: nil)
+    .media-body
+      = render partial: 'request_history_heading', locals: { creator: bs_request.creator,
+                                                             created_at: bs_request.created_at }
+      created request
+      = render partial: 'request_history_body', locals: { text: bs_request.description }
+
+  - history.each do |element|
```

Comment:
IMO, this could be into a partial, but using `collection` instead of `each` [[1]](https://github.com/openSUSE/open-build-service/pull/6893/commits/e4ccc1a1a2572d0d35c8e4df30c07c7c86944822#r253272363)

Reply:
@DavidKang 

> IMO, this could be into a partial, but using collection instead of each [1]

mm..? :confused: you still need `each` if you use a collection.... :confused: